index.bs

```diff
@@ -4078,30 +4090,12 @@ Workshops and Symposia</h2>
 	This helps ensure that speakers and authors
 	have adequate time to prepare position papers and talks.
 
-<h2 id="Liaisons">
-Liaisons</h2>
-
-	W3C uses the term <dfn export>liaison</dfn>
-	to refer to coordination of activities with a variety of organizations,
-	through a number of mechanisms
-	ranging from very informal
-	(e.g., an individual from another organization participates in a W3C Working Group,
-	or just follows its work)
-	to mutual membership,
-	to even more formal agreements.
-	Liaisons are not meant to substitute for W3C membership.
-
-	All liaisons <em class="rfc2119">must</em> be coordinated by the [=Team=]
-	due to requirements for public communication;
-	patent,
-	copyright,
-	and other IPR policies;
-	confidentiality agreements;
-	and mutual membership agreements.
+<h2 id=mou oldids="Liaisons">
+Memorandum of Understanding</h2>
 
 	The [=W3C Director=] <em class="rfc2119">may</em> negotiate
 	a Memorandum of Understanding with another organization.
-	For the purposes of the W3C Process a <dfn id="mou" export lt="Memorandum of Understanding | Memoranda of Understanding | MoU">Memorandum of Understanding</dfn> (<abbr>MoU</abbr>)
+	For the purposes of the W3C Process a <dfn export lt="Memorandum of Understanding | Memoranda of Understanding | MoU">Memorandum of Understanding</dfn> (<abbr>MoU</abbr>)
```

Comment:
Don't break links. Keep the ID here.

Reply:
I have not deleted that id, but moved it to the section title, which only speaks of MoUs now. I can restore it to what it was, but that felt reasonable to me.